terraform/modules/gost_website/runtime_config.tf

```
@@ -2,7 +2,8 @@ locals {
   deployConfigContents = templatefile(
     "${path.module}/tpl/deploy-config.js",
     {
-      gost_api_domain = var.gost_api_domain,
+      gost_api_domain    = var.gost_api_domain,
+      gost_use_new_table = var.gost_use_new_table,
```

Note - I have _not_ tested TF locally. Will CI runs catch any mistakes or should I get my env setup for that?

---

CI runs will catch failures to run `terraform plan`, inspecting the plan output is always a good idea. In this specific case, I don't think that would catch a mistake involving a bad `var.gost_use_new_table` value or something like that, but I think it would be quite complicated to test this via ECS-in-LocalStack. Personally, I think it would be okay to wait for this to get deployed to Staging and just do manual testing there.

---

Well what do you know! It appears that CI _did_ [catch something useful](https://github.com/usdigitalresponse/usdr-gost/actions/runs/5360920348/jobs/9726372315?pr=1563#step:10:31) after all.

Looks like the template is expecting a variable named `use_new_table`, not `gost_use_new_table`.
```suggestion
      gost_api_domain = var.gost_api_domain,
      use_new_table   = var.gost_use_new_table,
```

---

I was pretty confused by this until I realized I renamed the variable locally everywhere but forgot to stage the js template 🤦‍♂️